### DISCUSSION TITLE
单位转换 -- Digital (存储) 是否需要支持1Kb = 1000b 的转换？

### DISCUSSION THREAD

**Discussion by ghost** [he3-app/he3-feedback#15] [General]
1000和1024没有统一的使用规则， 看是否需要支持下？

**Comment by zhenweiwang1990** [comment]
具体在工具的哪些场景，是希望同时显示两个标准的数字？正常来说计算机软件的标准都是 1024，计算机硬件标准是 1000。

**Reply by ghost** [reply]
增加一个select或者是Radio 能够选择1024 或者1000 就行。

**Comment by zhenweiwang1990** [comment]
我们考虑增加一个 Digital(1000 based) 这个单位类别，方便这种场景使用。
